scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1334,12 +1334,16 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
           </tr>
         </thead>
         <tbody>
+
         {% for acmg in ["P","LP", "VUS", "LB", "B", ] %}
-          {% for variant in variants|selectattr("acmg_classification.short", "equalto", acmg)|selectattr("first_rep_gene.hgnc_symbol", "defined")|sort(attribute='first_rep_gene.hgnc_symbol')%}
+          {% for variant in variants|selectattr("acmg_classification", "defined")|selectattr("acmg_classification.short", "equalto", acmg)|selectattr("first_rep_gene.hgnc_symbol", "defined")|sort(attribute='first_rep_gene.hgnc_symbol') %}
```

Comment:
This solution works, but you might want to trim the variant HGVS when it's too long. We had a dedicated PR not so long ago..

<img width="1305" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/36dc8f19-00fc-4049-8725-c82c57da090e">

Reply:
Good point; we should have like a `pretty_print for the report as well...